_posts/lisetted/2017-05-30-lisetted-clicky-turtle.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+author: lisetted
+title: "Lisette's Clicky Turtle"
+---
+
+trinket
+<iframe src="https://trinket.io/embed/python/47fd051be1" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Reflection:
+
+I wanted to create a guessing game where the user tries to guess what randomly generated shape will appear next. Instead, I have 3 boxes that the user must draw some shapes in, and one box where the shape is randomly generated. In the first 3 boxes, there are unique colors for the shape if drawn in the box, and red if it is out of the box. 
```

Comment:
Nice scope change here.  The guessing game sounds neat, but there would've been lots of parts not related to your user interface.